packages/cmsds/README.md

```diff
@@ -49,10 +50,14 @@ The design system follows a variation of [ITCSS](http://thomasbyttebier.be/blog/
 ├── fonts
 ├── images
 └── src                 Non-compiled Sass and JSX
-    ├── base            Base HTML styles
```

Comment:
👍

Reply:
This has moved down to `styles` now, right? I guess it says `etc...`, so it's not necessary to list it out explicitly 👍

Reply:
The contents of the `base` folder were moved into styles (`typography`, `icons`, `base`). I actually removed it `base` folder because I felt like it didnt do a great job of describing the type of  styles inside the folder, and also because theres a second `base` folder for the `ds-base` class. Thinking on it again, maybe it is worth having another layer of organization in `styles`, but I'm not sure what to call it.

Reply:
Heres what it looks like now: 
```
styles
        ├── settings          // variables, mixins, scss that doesnt output css basically
        ├── typography   // Namespaced element selector for typography (ds-h1, ds-text)
        ├── base               // Namespaced element selector for body (ds-base)
       └── etc...  
```
@line47 @pwolfert What do yall think? What should the folder be called that contains `typography` and `base`? Fyi `icon` will eventually be moved to components, it doesnt belong here really

Reply:
I think it's fine how you have it now. It was a good move to put it in the styles folder because it's only related to styles, and the folders just keep the styles folder clean

Reply:
Actually, here's another idea: What if the `utilities` folder moves inside styles too since it's only about utility CSS classes? And what if `base`, `icons`, and `typography` moves into utilities because they are actually utility classes?

Reply:
One issue, however, is that the docs site is organized to have them called out separately from the rest of the utilities. Do they need to be? Could we have another landing page for the new Design section that touches on those elements and links to their pages in the utilities section?

Reply:
Ah I see. I think the biggest distinction between `utilities` and `base`/`typography` is that `utilities` are applying specific CSS properties, while `base`/`typography` are generally for HTML element related styles like `h1`, `p`, `body` etc (`icons` will be moved to components because it will become a React component). This difference is the reason why there's a difference in naming convention, `ds-h1` vs  `ds-u-font-size--title`. 

I was thinking about how there are `scss` files in all 3 folders, so instead of organizing according to js vs scss, it might be better to organize it by the website hierarchy. I was considering how `component` and `utilities` are the two largest pieces of documentation content, so it might be better to keep them at the root.  Similarly, `typography`, `base` are under the same top level website category of ["Styles"](https://gyazo.com/97896682de061524168520a6d83523bd) (or "Design" in later versions)

I'm not really sure whats the best situation here... I wish it would be an easy typical css/js organization, but I also think modeling it off of the website structure can make it easier to find relevant files for new maintainers

Reply:
I'm thinking of it like  `base`, `icons` and `typography` are high-level items that impact setting up a child DS and are used in components like alerts, buttons, etc... while the utilities are more for modification or extending components of the design system. To me, that's the distinction as to why they are grouped the way they are. 🤔 
I'd also say this is why spacing and grid are separated as well.

Reply:
Oh right, I moved `grid` to utilities but I could see it going along with `spacing`, `typography` and `base` too...